spec.md

```diff
@@ -1009,6 +1022,9 @@ It cannot be connected to both a `UInt`{.firrtl} and an `AsyncReset`{.firrtl}.
 The `AsyncReset`{.firrtl} type can be connected to another
 `AsyncReset`{.firrtl} or to a `Reset`{.firrtl}.
 
+Two enumeration types are equivalent if both have the same number of variants,
+and both the enumeration's i'th variants have matching names and types.
```

Comment:
Mega nit: `enumeration types'`. Also, I'd change "matching names and types" to "matching names and equivalent types" as "matching" isn't defined on types. (I supremely nit that "matching names" should be "equal names" and make it clear that this is string equality, but this is not done for `Bundle` today.)

Reply:
Fixed.  I am a little worried because the types should exactly match with no trunc or ext ("An unsigned integer type is always equivalent to another unsigned integer type regardless of bit width"), similar to array types.  But the wording for arrays uses "equivalent types" as well, so it should be fine.